src/clr-angular/utils/drag-and-drop/providers/drag-and-drop-event-bus.spec.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {ClrDraggableEvent} from "../interfaces/draggable-event";
+
+import {ClrDragAndDropEventBus} from "./drag-and-drop-event-bus";
+
+export default function(): void {
+    describe("Drag And Drop Event Bus Provider", function() {
+        beforeEach(function() {
+            this.dndEventBus = new ClrDragAndDropEventBus();
+        });
+
+        it("notifyAllOnDragStart() should cast to dragStart subject", function() {
+            const testPayLoad: ClrDraggableEvent = {dragDataStore: "dragStartTest"};
+            this.dndEventBus.dragStarted.subscribe((value: any) => {
+                expect(value).toBe(testPayLoad);
+                expect(value.dragDataStore).toBe("dragStartTest");
+            });
+            this.dndEventBus.notifyAllOnDragStart(testPayLoad);
```

Comment:
This test passes even if the Observable never emits anything. You can try it by just commenting out
```typescript
// this.dndEventBus.notifyAllOnDragStart(testPayLoad);
```
you'll see the test still passes.

What you want is to set some variable in the subscription, and check that this variable was properly set on line 24, right after the call to `notify...()`.